docs/relay/nodes.rst

```diff
@@ -14,12 +14,10 @@ Example usage (taken from the `Starwars Relay example`_):
 
 .. code:: python
 
-    class Ship(graphene.ObjectType):
-        '''A ship in the Star Wars saga'''
-        class Meta:
-            interfaces = (relay.Node, )
+    class Ship(ObjectType, interfaces=(relay.Node,)):
```

Comment:
Do you prefer this way of supplying meta args? I've not used it much myself since I'm used to Meta inner class with Django.

Reply:
I've not used this way before either but it's quite nice. I guess since we're dropping Python 2 support we should just go all out on Python 3 features.

Reply:
@dvndrsn  I like this for `graphene`, as it feels more pythonic, with the Meta perhaps having it's place in `graphene-django`. 

It's offered in `UPGRADE-v2.0.md` under `Meta as Class arguments`, which may highlight it as best practice to new users/upgrading folks.

Also, `class Meta` seems to offer some confusion among users, if SO is any indication:

https://stackoverflow.com/search?q=%5Bgraphene-python%5D+class+Meta

Reply:
Yeah, that's true. Especially with Graphene-Django and Graphene-SQLAlchemy. Both of those projects have a lot of use of `Meta` parameters.

Reply:
I feel like docs should at least present both approaches. As I tend to use `Meta` anyway in my Django project, where `graphene-django` is also used, it will be nice to have a clear indication that you can do that with `graphene` also. That may not be clear for folks less familiar with Python.